kpops/component_handlers/streams_bootstrap/app_handler.py

```diff
@@ -21,35 +20,52 @@
 
 
 class AppHandler:
-    def __init__(self, helm_wrapper: HelmWrapper):
-        self._helm_wrapper = helm_wrapper
-        helm_wrapper.helm_repo_add()
+    def __init__(
+        self,
+        helm_config: HelmConfig,
+        helm_repo_config: HelmRepoConfig,
+        helm_diff: HelmDiff,
+    ):
+        self._helm_wrapper = Helm(helm_config)
+        self.repository_name = helm_repo_config.repository_name
+        self.chart_version = helm_repo_config.version
+        self._helm_wrapper.add_repo(
+            helm_repo_config.repository_name,
+            helm_repo_config.url,
+            RepoAuthFlags(
+                username=helm_repo_config.username, password=helm_repo_config.password
+            ),
+        )
+        self.helm_diff = helm_diff
 
     def install_app(
         self,
         release_name: str,
+        application_type: str,
         namespace: str,
         values: dict,
-        app_type: ApplicationType,
         dry_run: bool,
-        local_chart_path: Path | None = None,
```

Comment:
Hm, we used that before. I think we should still make it possible to use that. I would even say we should just pass through the chart parameter